pkg/serviceproxy/service_proxy.go

```diff
@@ -151,29 +167,8 @@ func (s *serviceProxy) validate() error {
 	if s.key == "" {
 		return fmt.Errorf("key is required")
 	}
-	if s.apiserverCA == "" {
-		return fmt.Errorf("apiserver-ca-cert is required")
-	}
 	if s.ocpserviceCA == "" {
```

Comment:
is this check still needed?

Reply:
Not really, since this value is optinal.

Reply:
Fixed.